Source/WorkersES6/decodeDraco.js

```diff
@@ -276,12 +276,12 @@ function decodePrimitive(parameters) {
   var dracoDecoder = new draco.Decoder();
 
   // Skip all parameter types except generic
-  var attributesToSkip = ["POSITION", "NORMAL", "COLOR", "TEX_COORD"];
-  if (parameters.dequantizeInShader) {
-    for (var i = 0; i < attributesToSkip.length; ++i) {
-      dracoDecoder.SkipAttributeTransform(draco[attributesToSkip[i]]);
-    }
-  }
+  // var attributesToSkip = ["POSITION", "NORMAL", "COLOR", "TEX_COORD"];
+  // if (parameters.dequantizeInShader) {
+  //   for (var i = 0; i < attributesToSkip.length; ++i) {
+  //     dracoDecoder.SkipAttributeTransform(draco[attributesToSkip[i]]);
+  //   }
+  // }
```

Comment:
To help keep the code clean let's delete this code and failing tests in a single isolated commit, then in a new issue mention that the commit should be reverted once the draco issue is fixed upstream and we update the npm package.